examples/gitlab-asg/Makefile

```diff
@@ -5,29 +5,24 @@
 ## Use envsubst and vars.env to render the json config files for cfssl
 render-tls-configs:
 	@. ./vars.env && envsubst <ca-csr.tpl > ca-csr.json
-	@. ./vars.env && envsubst <server-csr.tpl > server-csr.json
+	@. ./vars.env && envsubst <cert-csr.tpl > cert-csr.json
 
 ## Use envsubst and project.env to render terraform-tfvars.tpl into terraform.tfvars
 render-tfvars:
 	@. ./vars.env && envsubst <terraform-tfvars.tpl > terraform.tfvars
 
-## Use envsubst and project.env to render terraform-tfvars.tpl into terraform.tfvars
-render-ssh-config:
-	@. ./vars.env && envsubst <ssh_config.tpl
-
 ## Use ssh-keygen to generate a keypair for SSH
 generate-ssh-key:
 	@ssh-keygen -q -N "" -b 4096 -C "test" -f ./id_rsa
 
 ## write out a tfvars file for a test deploy
 generate-tfvars:
-	@echo "ssh_pubkey = \"./id_rsa.pub\"" > terraform.tfvars
-	@echo "region = \"us-east-2\"" >> terraform.tfvars
-
+	@echo "ssh_pubkey = \"./id_rsa.pub\"" >> terraform.tfvars
```

Comment:
Any reason this can't be part of `make render-tfvars`, and then change the README to have you run `make render-tfvars` later in the process (e.g. just before `make network`)?  I don't think any of the earlier steps depend on `terraform.tfvars` existing.

Reply:
Great catch! These two steps were added later and layered on and I didnt notice you could easily move the `render_tfvars` and consolidate with the other two modifications.

Fixed and tested.